index.js

```diff
@@ -1272,6 +1315,8 @@
     ap: ap,
     lift2: lift2,
     lift3: lift3,
+    apFirst: apFirst,
+    apSecond: apSecond,
```

Comment:
I think you can still have access to lift2 if you move apFirst because of js hoisting